src/components/ChartTypes.jsx

```diff
@@ -5,6 +5,8 @@ import { SettingIcon } from './Icons.jsx';
 import Tooltip from './Tooltip.jsx';
 import '../../sass/components/chart-types.scss';
 
+const TypeIcon = ({ Icon, props }) => (<Icon {...props} />);
```

Comment:
Why do we need this wrapper? Is it for better naming?

Reply:
@balakrishna-binary  just for beauty :sweat_smile:  I just fixed it